ext/native-decls/SetWeaponsNoAutoSwap.md

```diff
@@ -11,4 +11,8 @@ void SET_WEAPONS_NO_AUTOSWAP(BOOL state);
 Disables autoswapping to another weapon when the current weapon runs out of ammo.
 
 ## Parameters
-* **state**: On/Off
\ No newline at end of file
+* **state**: On/Off
+
+```
+This native won't have any affect if setting flag 48 to true, using SetPedConfigFlag native.
```

Comment:
Adding descriptions below 'Parameters' won't show at all. Also, why is this in code blocks?